src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java

```diff
@@ -12,9 +12,14 @@ public static void printError() {
     }
 
     public static void addFormatError() {
-        printer.showToUser("Wrong format, please enter in the format: \n"
-                + "\t add [NAME_OF_MOVE] /n [NUMBER_OF_REPETITIONS] /w "
-                + "[WEIGHT]");
+        printer.showToUser("Wrong format, please enter in the format: \n\t "
+                + "add [NAME_OF_MOVE] /n [NUMBER_OF_REPETITIONS] /w [WEIGHT]");
+    }
+
+    public static void addFormatNegativeError() {
+        printer.showToUser("Wrong format, please enter in the format: \n\t "
+                + "add [NAME_OF_MOVE] /n [NUMBER_OF_REPETITIONS] /w [WEIGHT]\n\t "
+                + "Please make sure [NUMBER_OF_REPETITIONS] and [WEIGHT] are positive numbers.");
```

Comment:
Should this be "Please make sure ... are non-negative numbers" since move like push-up can have weight of 0?

Reply:
Great catch